google/cloud/firestore_v1/async_transaction.py

```diff
@@ -148,7 +148,7 @@ async def get_all(self, references):
             .DocumentSnapshot: The next document snapshot that fulfills the
             query, or :data:`None` if the document does not exist.
         """
-        return self._client.get_all(references, transaction=self)
+        return await self._client.get_all(references, transaction=self)
```

Comment:
It appears that this pr would change the return type from a coroutine to the result. Was this a bug?

Reply:
Yep, I hadn't realized that the rpc call in the microgen client returned a coroutine.